src/libs/runtime-utils.js

```diff
@@ -69,6 +69,11 @@ export const runtimeCost = ({ runtimeConfig, status }) => {
   }
 }
 
+export const getGalaxyCost = app => {
+  return hourlyKubernetesAppCost(app || { kubernetesRuntimeConfig: { machineType: 'n1-standard-8' } }) +
+    persistentDiskCost({ size: 250 + 10 + 100, status: 'Running' }) // 250Gb for the NFS disk, 10Gb for the postgres disk, and 100Gb for the boot disk
```

Comment:
why do we pass in a hardcoded 'Running' here?

Reply:
I believe the motivation is so that we are be able to show the user what the cost _will_ be when running, otherwise it could show as $0.00